Revert/Sources/HomeViewController.swift

```diff
@@ -63,8 +59,7 @@ final class HomeViewController: UITableViewController {
   private var collection = CollectableCollection<HomeItem>(items: .Home)
   private var dataSource: DataSource<HomeItem, HomeCell>
   private var isSplitViewControllerCollapsed: Bool {
-    // If self.splitViewController is nil then we're on iOS7 on phone, i.e equivalent to collapsed mode
-    return self.splitViewController?.backwardCompatibleCollapsed ?? true
+    return self.splitViewController?.collapsed ?? true
   }
```

Comment:
Yeah, `splitViewController` is still an optional in the API though, so it can be `.None`. I think you're right about the logic of this check though — does a non-existent split view controller still mean it's collapsed under iOS > 7?